circuits/lib/utils/idUtils.circom

```diff
@@ -0,0 +1,169 @@
+pragma circom 2.0.9;
+
+include "../../../node_modules/circomlib/circuits/bitify.circom";
+include "../../../node_modules/circomlib/circuits/binsum.circom";
+include "../../../node_modules/circomlib/circuits/eddsaposeidon.circom";
+
+template ProfileID(){
+    signal input in;
+    signal input nonce;
+    signal output out;
+
+    component hash = Poseidon(2);
+    hash.inputs[0] <== in;
+    hash.inputs[1] <== nonce;
+
+    component genesis = TakeNBits(27*8);
+    genesis.in <== hash.out;
+
+    component oldIdParts = SplitID();
```

Comment:
Why the name is `oldIdParts`? Sounds confusing or I missed something...

Reply:
renamed it do genesisIDParts